.github/CODEOWNERS

```diff
@@ -75,3 +75,10 @@ subprojects/docs/src/docs/userguide/dep-man/
 subprojects/docs/src/docs/userguide/jvm/                                          @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/reference/directory_layout.adoc               @gradle/bt-jvm
 subprojects/docs/src/docs/userguide/troubleshooting/version_catalog_problems.adoc @gradle/bt-jvm
+
+# Dev-prod team
+build-logic/                                @gradle/bt-developer-productivity
```

Comment:
🤔 There are some things under `gradle` that we might want to add. Especially `mirrors.settings.gradle.kts`.